schema/table.go

```diff
@@ -248,55 +281,50 @@ func (t *Table) ValidateName() error {
 	return nil
 }
 
-// GetAddedColumns returns a list of columns that are in this table but not in the other table.
-func (t *Table) GetAddedColumns(other *Table) []Column {
-	var added []Column
-	for _, c := range t.Columns {
-		if other.Columns.Get(c.Name) == nil {
-			added = append(added, c)
-		}
-	}
-	return added
-}
+// added columns with no constraints - migrate // table.GetAddedColumns(other)
+// added columns with not null - drop table // table.GetAddedConstraintColumns(other)
+// added columns with unique - drop table
 
-// GetChangedColumns returns a list of columns that are in this table but have different type in the other table.
-// returns got, want
-func (t *Table) GetChangedColumns(other *Table) (got ColumnList, want ColumnList) {
+// changed columns: change type - drop table or drop/add column? // table.GetChangedColumns(other)
+// changed columns: added constraint not null or unique - drop table // table.GetChangedColumnsWithAddedConstraints
+// changed columns: removed constraint: not null, removed unique - drop constraint? do nothing? what if a user is adding a unique constraint? we will drop it? // table.GetChangedColumnsWithRemovedConstraints
+
+// removed columns: with no constraints - do nothing?
+// removed columns: with constraints not null or unique - drop column? drop table ? // table.RemovedColumnsWithConstraints(other)
+
+// changed pks: drop table // table.IsPrimaryKeyEqual(other)
+
+func (t *Table) GetChanges(other *Table) []TableColumnChange {
```

Comment:
```suggestion
func (new *Table) GetChanges(old *Table) []TableColumnChange {
```

Maybe this will make it easier to follow? Non blocking

Reply:
`new` being a reserved word probably can't be used, but the `old` change is 👍

Reply:
Fixed.